frameworks/cassandra/src/main/java/com/mesosphere/sdk/cassandra/scheduler/Main.java

```diff
@@ -38,35 +25,21 @@ public static void main(String[] args) throws Exception {
             throws Exception {
         SchedulerFlags schedulerFlags = SchedulerFlags.fromEnv();
         RawServiceSpec rawServiceSpec = RawServiceSpec.newBuilder(pathToYamlSpecification).build();
-        List<String> localSeeds = getLocalSeeds(rawServiceSpec.getName());
         DefaultScheduler.Builder schedulerBuilder = DefaultScheduler.newBuilder(
                 DefaultServiceSpec.newGenerator(rawServiceSpec, schedulerFlags)
-                        .setAllPodsEnv("LOCAL_SEEDS", Joiner.on(',').join(localSeeds))
                         .build(),
                 schedulerFlags)
                 .setPlansFrom(rawServiceSpec)
-                .setCustomResources(getResources(localSeeds))
                 .setRecoveryManagerFactory(new CassandraRecoveryPlanOverriderFactory());
-        return schedulerBuilder;
-    }
+        schedulerBuilder.setCustomResources(getResources(schedulerBuilder.getStateStore()));
```

Comment:
Should continue using the builder chain IMO, could even just return the result of the chain without a `schedulerBuilder` variable